src/expr.rs

```diff
@@ -666,7 +667,8 @@ fn rewrite_match(context: &RewriteContext,
         if !missed_str.is_empty() {
             result.push('\n');
             result.push_str(&arm_indent_str);
-            result.push_str(missed_str);
+            result.push_str(&rewrite_comment(&missed_str, false,
+                                             width, arm_indent, context.config));
```

Comment:
Nit: one arg per line

Slightly worried that we don't reformat this in cargo test.